app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -126,6 +126,17 @@ data class Patient(
         newUpdatedAt: Instant
     )
 
+    @Query("""
+      UPDATE Patient
+      SET recordedAt = :bpRecordedAt, syncStatus = :pendingStatus, updatedAt = :updatedAt
+      WHERE uuid = :patientUuid AND recordedAt > :bpRecordedAt
+    """)
+    abstract fun updateRecordedAt(
```

Comment:
There is nothing blood pressure related about this query; we are just setting the `recordedAt` of the patient to the minimum of the patient's `recordedAt` and the BP's `recordedAt`. Should we rename the method and the `bpRecordedAt` parameter to better reflect this?

Reply:
This goes for the method and parameter naming in the `PatientRepository` as well since the `Instant` could be from anywhere. It's only the `BloodPressureEntrySheetController` that decides that the `recordedAt` of the BP could potentially be the new `recordedAt` of the patient.

Reply:
Good point. I will rename the parameter.
why do I need to rename the method though?

Reply:
This is just my opinion, but when I read the name `updateRecordedAt`, I feel it will always update the `recordedAt` of the patient. But this method conditionally updates it.

Reply:
Done.